robottelo/ssh.py

```diff
@@ -237,7 +243,7 @@ def download_file(remote_file, local_file=None, hostname=None):
 
 
 def command(cmd, hostname=None, output_format=None, username=None,
-            password=None, key_filename=None, timeout=10):
+            password=None, key_filename=None, timeout=120, conn_timeout=10):
```

Comment:
I'm not sure whether 120 secs will be enough. API is returning results immediately, but for some requests which schedule tasks we're waiting for 300 sec to finish by default. The thing is CLI behavior is different and usually we don't get a result immediately but only after corresponding task has finished. 
I can see 2 possible approaches here: 
1) Set default timeout to 300 secs, so max timeout will be the same as in API, but we may start waiting too much for 'atomic' commands to finish (in case they stop responding, which is very unlikely though);
2) Leave 120 secs but set custom 300 secs for all the commands which are scheduling tasks and taking longer to finish, but the tricky part here is to form a list of such commands.